site/e2e/helpers.ts

```diff
@@ -191,31 +203,47 @@ export const createTemplate = async (
 		});
 	}
 
+	// If the organization picker is present on the page, select the default
+	// organization.
+	const orgPicker = page.getByLabel("Belongs to *");
```

Comment:
This may need to be updated when the orgs redesign is implementred

Reply:
it's fine. there's tons of stuff like that.